### ISSUE TITLE
Configuração de Firewall

### ISSUE THREAD

**Issue by rzrlew** [Morierti/GRS#25]
Todas as máquinas do interior da rede devem ter acesso ao exterior. Do exterior não deve existir acesso ás máquinas privadas. Aos servidores deve ser facultado o acesso com protocolos dos serviços que elas oferecem(HTTP, IMAP, POP, DNS, SSH, etc...).

**Comment by rzrlew** [comment]
Firewall está quase feita. Falta para os endereços privados com masquerade. Para acesso ao lawserver é preciso NAT?

**Comment by Morierti** [comment]
Penso que não, pelo que o servidor tem um endereço public por isso em principio não é necessário NAT.

**Comment by rzrlew** [comment]
Sim, mas os IP's privados dentro da autocar precisam de acesso ao lawServer? E se sim, é suficiente ter a rota indicada no routerISP? ou também devemos considerar o routerISP  do lado de fora da rede e meterter protecção da firewall...

**Comment by Morierti** [comment]
O routerISP penso que seja o border router e como tal deve permitir acesso das redes da autocar unicamente ao lawServer e vice-versa sendo que tudo o resto fica bloqueado pela firewall, mas também não tenho a certeza.

**Comment by Morierti** [comment]
Os servidores públicos (primary e secondary) não são acessíveis a partir do adminPC, porque tanto a VPN como o site www.autocar.ttt não abrem se fizer um pedido a partir do adminPC.
